roles/edpm_neutron_metadata/defaults/main.yml

```diff
@@ -30,7 +30,7 @@ edpm_neutron_metadata_agent_sidecar_haproxy_image_name: "{{ edpm_neutron_metadat
 
 # Neutron conf
 # DEFAULT
-edpm_neutron_metadata_agent_DEFAULT_host: '{{ ansible_facts["nodename"] }}'  # also in missing vars
+edpm_neutron_metadata_agent_DEFAULT_host: '{{ canonical_hostname }}'  # also in missing vars
```

Comment:
this is actually not used anywhere and I proposed cleanup in https://github.com/openstack-k8s-operators/edpm-ansible/pull/604

Reply:
OK then I remove the change from this role fully as based on #604 if the edpm_ovn role sets the proper hostname then metadata will pick that up automatically.

Reply:
done